src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -35,7 +36,27 @@ public class RestoreCommand
 
         public Guid ParentId { get; }
 
-        public const string ProjectRestoreInformation = "ProjectRestoreInformation";
+        // names for ProjectRestoreInformation and intervals
+        private static readonly string ProjectRestoreInformation = nameof(ProjectRestoreInformation);
```

Comment:
ideally these key names should be constants so that a simple refactoring wouldn't change the telemetry event schema. I know this is our current pattern across telemetry events but I'll encourage to not repeat for any new data or event. I already logged a separate bug to change it for existing events. NuGet/Home#6683